submissions/vakulinina/friends-app/main.js

```diff
@@ -0,0 +1,173 @@
+const minAge = 0;
+const maxAge = 99;
```

Comment:
`MIN_AGE` and `MAX_AGE`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage